tenprintcover.py

```diff
@@ -32,6 +32,9 @@
 
 import cairocffi as cairo
 
+PY2 = sys.version_info[0] == 2
+if PY2:
+    FileNotFoundError = IOError
```

Comment:
As of commit https://github.com/mgiraldo/tenprintcover-py/commit/7cc2fc19b4cee6928c5e012eed15462ffb1e46b2 this isn’t necessary anymore. If you prefer to keep this change locally, that’s alright, but I don’t think we should check the code into the repo.